ui/imports/shared/stores/BIP39_en.qml

```diff
@@ -1,15 +1,19 @@
 import QtQuick 2.13
 
 ListModel {
+    id: root
+
+    property var words: []
+
     Component.onCompleted: {
         var xhr = new XMLHttpRequest();
         xhr.open("GET", "english.txt");
```

Comment:
I know this is existing code but this should really move to backend code; QML even warns you in 5.15 that support for reading local files is dangerous and will be removed in Qt 6

Reply:
I agree